src/tutorial/source.rst

```diff
@@ -0,0 +1,83 @@
+.. py:currentmodule:: palace
+
+Source manipulation
+===================
+
+We have created a source in the last section.
+As said previously, its properties can be manipulated
+to create wanted effects.
+
+Moving the source
+-----------------
+
+Changing :py:meth:`Source.position` is one of the most noticeable,
+but first, we have to enable spatialization via :py:attr:`Source.spatialize`.
```

Comment:
pos ain't no meth lol, and it should be explained that spatialize is only needed for positional rendering of non-mono resources (see the docs)

Reply:
As I understand the doc, `spatialize` is activated for non-mono resources by default?

> or *None* (spatialization is enabled based on playing a mono sound or not, default).

Is earphone a non-mono resource? I remember once I forgot to activate `spatialize` and moving source has no effect.

Reply:
Reading it again it doesn't sound really clear, but what is supposed to mean is
* Mono audio are always rendered positionally
* By default (`spatialize == None`), positional setting (position, velocity, orientation) on nonmono audio have no effect

Mono audios are those with only one channel.  Note that sources in alure terminology are where and how those audios are played.